its/ruling/src/test/resources/sources/build.gradle

```diff
@@ -3,6 +3,6 @@ plugins {
 }
 
 golang {
```

Comment:
Is this file needed at all? The `gradle tasks` shows me the following error:
```
Project directory '.../sonar-go/its/ruling/src/test/resources/sources/go' is not part of the build defined by settings file '.../sonar-go/settings.gradle'. If this is an unrelated build, it must have its own settings file.
```

Maybe it should be removed?

Reply:
Yes, I can see that it exists from the very beginning of the Go analyzer without a clear usage. And the `gogradle` plugin is archived for two years already. I'll remove the file